feed/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS-FRAMEWORKS | Feed</title>
+    <meta name="description" content="On this page you can post a new Twist and read content aimed for your liking.">
+    <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../dist/css/styles.css">
+    <link rel="stylesheet" href="../dist/css/custom-header.css">
+</head>
+<body class="container-fluid text-center bg-image p-0">
+    <header class="container-fluid pt-3 pb-4 custom-header">
+        <div class="row justify-content-center">
+            <div class="col-12 col-md-8">
+                <div class="row">
+                    <div class="col-4 d-flex align-items-center">
+                        <a href="../profile/index.html"><img class="img-fluid d-md-none" src="../images/Profile-icon.png" alt="Twister Logo"></a>
+                        <div class="d-none d-md-block"><a href="../profile/index.html"><h4>Profile</h4></a>.</div>
+                    </div>  
+                    <div class="col-4">
+                        <h1 class="h4 font-weight-normal">Feed</h1>
+                    </div> 
+                    <div class="col-4"></div> 
+                </div>
+            </div>
+        </div>
+        <div class="row justify-content-center mt-4">
+            <div class="col-12 col-md-8">
+                <div class="card">
+                    <div class="card-body p-0">
+                        <div class="form-group">
+                            <textarea class="form-control" rows="3" placeholder="Create Twist..."></textarea>
+                        </div>
+                        <div class="d-flex mb-2" data-toggle="buttons">
+                            <input type="radio" class="btn-check" name="postType" id="joke" autocomplete="off">
+                            <label class="col-2 px-1 py-1 ms-2 btn btn-outline-primary" for="joke">Joke</label>
+                            <input type="radio" class="btn-check" name="postType" id="riddle" autocomplete="off">
+                            <label class="col-2 px-1 py-1 ms-1 btn btn-outline-primary" for="riddle">Riddle</label>
+                            <input type="radio" class="btn-check" name="postType" id="quote" autocomplete="off">
+                            <label class="col-2 px-1 py-1 ms-1 btn btn-outline-primary" for="quote">Quote</label>
+                            <button type="button" class="col-4 px-1 py-1 ms-auto me-2 btn btn-primary"><img src="../images/Twister.png" alt="Twister icon"> Publish</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container-fluid mt-3" style="max-height: calc(100vh - 20%); overflow-y: auto;">
```

Comment:
```suggestion
    <main class="container-fluid mt-3">
```

There is a much better way to do this. Setting `min-vh-100 d-flex flex-column` on the body element, then `flex-grow-1` on the main element.